src/screens/home/Home.tsx

```diff
@@ -253,7 +253,9 @@ export default class Home extends BaseScreen<Props, State> {
                   <CardItem
                     style={cardStyle.cardItem}
                     button
-                    onPress={() => navigation.navigate('UsersNavigator', { key: `${Utils.randomNumber()}` })}>
+                    onPress={() =>
+                      navigation.navigate('UsersNavigator', { params: { userIDs: undefined }, key: `${Utils.randomNumber()}` })
```

Comment:
No need to pass `params: { userIDs: undefined }`

Reply:
@LucasBrazi06 
It's necessary otherwise the Users list will keep showing the last visited car driver via the Home menu. 
For the sidebar access to the users, we did not find a way to reload all the users